MockKit/MockCGMDataSource.swift

```diff
@@ -146,6 +147,7 @@ extension MockCGMDataSource.Model: RawRepresentable {
         case sineCurve = "sineCurve"
         case noData = "noData"
         case signalLoss = "signalLoss"
+        case unreliableData = "unreliableData"
```

Comment:
[nit] You actually don't need any of the `= "..."` here. When you declare an `enum` of type `String` the default `rawValue` for each case is the same as the enum name (capitalization and everything). Since the enum name and string values are exactly the same here it can be dropped.